.github/workflows/check_pr.yml

```diff
@@ -62,6 +64,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1.3
 
+      - name: Install sudo package
+        run: apt update && apt install sudo
+
+      - name: Start Solana Local Validator
+        uses: switchboard-xyz/solana-local-validator@v0.1
```

Comment:
Does it mean that we can move away from docker action?

Reply:
Yes since we cannot load solana programs using the docker action

Reply:
Created https://github.com/espresso-cash/espresso-cash-public/pull/907 for this. Also apparently sudo installation is not needed.